apps/src/templates/projects/deleteDialog/DeleteProjectDialog.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { Component, PropTypes } from 'react';
+import BaseDialog from '../../BaseDialog';
+import DialogFooter from '../../teacherDashboard/DialogFooter';
+import Button from '../../Button';
+import i18n from '@cdo/locale';
+
+const styles = {
+  dialog: {
+    paddingLeft: 20,
+    paddingRight: 20,
+    paddingBottom: 20,
+  },
+};
+
+export default class DeleteProjectDialog extends Component {
+  static propTypes = {
+    isOpen: PropTypes.bool.isRequired,
+    projectId: PropTypes.string,
+    isDeletePending: PropTypes.bool,
+    onClose: PropTypes.func,
```

Comment:
Should the onClose prop be required?

Reply:
I think so, and projectId should probably also be required. I'll update in the follow up when I connect this dialog to the table.